### PR TITLE
Swipe to reply

### PR DESCRIPTION
**Pull request by EvGen94**
Add swipe to reply feature 
In order to perform a swipe to reply a new method has been added to the messages cubit

### CODE REVIEW

**Review by bobs4462 — APPROVED** [review]
Need to rework swipeThreadFetch method

**Review by bobs4462 — CHANGES_REQUESTED** [review]
Need to rework swipeThreadFetch method to be more optimized

File: lib/blocs/messages_cubit/messages_cubit.dart [hunk 1]
```diff
@@ -69,6 +69,20 @@ abstract class BaseMessagesCubit extends Cubit<MessagesState> {
     }
   }
 
+  Future<void> swipeReply(
+    String threadId,
+  ) async {
+    Message parentMessage;
+
+    parentMessage = await _repository.getMessage(messageId: threadId);
```
Comment (bobs4462) [inline lib/blocs/messages_cubit/messages_cubit.dart:77]:
```
final parentMessage = await _repository.getMessage(messageId: threadId);
```

**Review by bobs4462 — APPROVED** [review]

File: lib/blocs/messages_cubit/messages_cubit.dart (hunk 1 above)
Comment (bobs4462) [reply lib/blocs/messages_cubit/messages_cubit.dart:77]: I think that would be more professional :)

Comment (EvGen94) [reply lib/blocs/messages_cubit/messages_cubit.dart:77]: Yes, I will remove -  Message parentMessage;

**Pull request merged**